ktransw.py

```diff
@@ -312,7 +312,9 @@ def main():
             # TODO: we loose stdout/stderr interleaving here
             # TODO: the error messages refer to lines in the temporary,
             # preprocessed KAREL source file, not the original one.
-            sys.stdout.write(pstdout.replace(dname, os.path.dirname(kl_file)) + '\n')
+            pstdout = pstdout.decode('utf-8')
+            ktrans_out = pstdout.replace(dname, os.path.dirname(kl_file)) + '\n'
```

Comment:
```suggestion
            ktrans_out = pstdout.decode('utf-8').replace(dname, os.path.dirname(kl_file)) + '\n'
```

Could be nice to chain this (and then remove the `pstdout = pstdout.decode('utf-8')` line above).

Reply:
definitely. I think I was just testing and forgot to get rid of the extra line.